### ISSUE TITLE
configure accepts --destdir and --libdir, but they don't actually work

### ISSUE THREAD

**Issue by yurivict** [janestreet/js-build-tools#4]
Makefile still installs into $PREFIX and ignores $DESTDIR. It installs into /usr/local/lib ignoring the supplied --libdir.

**Comment by ghost** [comment]
This package is no longer maintained. It is recommended to switch to [jbuilder](http://github.com/janestreet/jbuilder) instead

**Comment by yurivict** [comment]
One project that I am trying to run uses the "async" module. "async" in turn requires 
"typerep" (https://github.com/janestreet/typerep). "typerep" requires "js-build-tools":
```
ocamlfind: Package `js-build-tools.ocamlbuild_goodies' not found
```
Do you know what is wrong? Why does "async" module ultimately require the discontinued module "js-build-tools"?

**Comment by ghost** [comment]
The latest released version of async and typerep (v0.9) do not require js-build-tools.
